lua_modules/gossip/gossip.lua

```diff
@@ -0,0 +1,268 @@
+-- Gossip protocol implementation
+-- https://github.com/alexandruantochi/
+
+local gossip = {};
+local constants = {};
+local utils = {};
+local network = {};
+local state = {};
+
+-- Utils
+
+utils.debug = function(message)
+    if gossip.config.debug then
```

Comment:
I think the coding style is 2 space indentation. Couldn't find a counter sample but I am not sure if it is documented.